homeassistant/components/ads/__init__.py

```diff
@@ -217,7 +226,7 @@ def add_device_notification(self, name, plc_datatype, callback):
             else:
                 hnotify = int(hnotify)
                 self._notification_items[hnotify] = NotificationItem(
-                    hnotify, huser, name, plc_datatype, callback
+                    hnotify, huser, name, plc_datatype,  callback
```

Comment:
Looks like double space after the comma. Please make sure to run black code formatter.

Reply:
I had fix it. And now i activate "black formatter". So for the future, ever code ist "black formatted"